### ISSUE TITLE
Server build is not compatible with Java 17

### ISSUE THREAD

**Issue by ndoschek** [eclipse-emfcloud/coffee-editor#444]
Building the coffee-editor backend with Java 17, errors with the following output:

```
$ ./run.sh -b
[16:02:34.550] Evaluate Options... 
  Build Backend (-b)
  Do not copy Backend (-c)
  Do not remove yarn.lock  (-ff)
  Do not build Frontend (-f)
  Do not run Frontend (-r)
[16:02:34.554] Build backend products
[ERROR] Error executing Maven.
[ERROR] java.lang.IllegalStateException: Unable to load cache item
[ERROR] Caused by: Unable to load cache item
[ERROR] Caused by: Could not initialize class com.google.inject.internal.cglib.core.$MethodWrapper
```

**Comment by tortmayr** [comment]
I had a quick look at this.
Looks like we need to update the Tycho Version (to 2.7) and
Update Xtext to 2.27 to properly support a Java 17 build.
Afaik Xtext. 2.27 should be available with the next Eclipse Release (2022-06).

**Comment by ndoschek** [comment]
Fix was reverted with #477 as workflow-dsl editor did not work after the update
